### ISSUE TITLE
Enable Uniswap v3 limit order bots with expiration dates and a schedule of limit orders

### ISSUE THREAD

**Issue by verabehr** [VolumeFi/paloma#1589]
As a user I want to be able to open up limit orders that have an expiration date and time as well as to open up multiple future limit orders when current limit orders expire unfilled.

**Acceptance criteria**
.1. Add the ability to automatically close a limit order after certain amount of time
    1. Hours
    2. Minutes
    3. Days
2. Add the ability to open a new limit order bot once the old one is closed at a new price with a new duration

https://etherscan.io/address/0x257facb5d1ffd0f7bb0ee2091bb44ab6018c8a8b'

**Comment by verabehr** [comment]
Blocked by https://github.com/VolumeFi/paloma/issues/1670

**Comment by verabehr** [comment]
new contract: 0xF545c98E8Bfd698B2EF4E60d797245Df7bA89C76

**Comment by verabehr** [comment]
this can be tested again. Contract was updated and backend was updated

**Comment by verabehr** [comment]
Blocked by https://github.com/VolumeFi/paloma/issues/1762 and https://github.com/VolumeFi/paloma/issues/1765

**Comment by taariq** [comment]
Closing as won't fix.
